src/common/reducers/select-repositories-form.js

```diff
@@ -0,0 +1,30 @@
+import * as ActionTypes from '../actions/select-repositories-form';
+import { hasRepository } from '../helpers/repositories';
+
+const initialState = {
+  selectedRepos: []
+};
+
+export function selectRepositoriesForm(state = initialState, action) {
+  switch(action.type) {
+    case ActionTypes.TOGGLE_REPOSITORY:
+      if (hasRepository(state.selectedRepos, action.payload)) {
+        return {
+          ...state,
+          selectedRepos: state.selectedRepos.filter(
+            (repo) => repo.fullName !== action.payload.fullName
```

Comment:
My 2c is that anything more complex than a single expression return should be cuddled, for clarity and the avoidance of mistakes down the line.